package.json

```diff
@@ -20,7 +20,9 @@
     "url": "https://github.com/LLK/scratchjr-website/issues"
   },
   "homepage": "https://github.com/LLK/scratchjr-website",
-  "dependencies": {},
+  "dependencies": {
+    "lint": "^0.7.0"
```

Comment:
Please remove this - eslint is already in the devDependencies, so this is unnecessary.